### DISCUSSION TITLE
Not able to flash Zephyr RTOS on CC3220SF LaunchXL

### DISCUSSION THREAD

**Discussion by Shubham-RY** [zephyrproject-rtos/zephyr#63513] [Q&A]
After following the steps mentioned [here](https://docs.zephyrproject.org/latest/develop/getting_started/index.html) I tried flashing the Blinky exmple on CC3220SF launchXL board. After the last command of _west flash_ I get the folowing output

![3e1cffa3-2878-4423-bf00-6d3c6fe4c900](https://github.com/zephyrproject-rtos/zephyr/assets/128672681/4325ba63-d7d8-4834-8790-bae6b55fcc90)

I don't see any LED blinking on the board. Is the output log that I am getting the expected result? Also do I need to perform some additional steps to get Zephyr running on my board?

I have used this command to build the program. 
_west build -p always -b cc3220sf_launchxl samples\basic\blinky_
I have the SOP jumper setting as 001 while flashing. Before flashing I have also tried flashing an empty project through Uniflash with SOP jumper settings for that being 010 as was recommended on one of the other [pages](https://readthedocs.amarulasolutions.com/docs/wiki/en/latest/zephyr/ti/CC3220SF.html#environment) I was referring to.

**Comment by Psychesnet** [comment]
Hi @Shubham-RY 

I use same device to try, but I have different error. I followed doc and switch SOP to try. Do you have any idea to fix this error? Thanks.

```
(.venv) eric@Erics-MBP zephyr % west flash
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner openocd
-- runners.openocd: Flashing file: /Users/eric/zephyrproject/zephyr/build/zephyr/zephyr.hex
Open On-Chip Debugger 0.12.0
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
ocd_process_reset_inner
Info : XDS110: connected
Info : XDS110: vid/pid = 0451/bef3
Info : XDS110: firmware version = 3.0.0.28
Info : XDS110: hardware version = 0x0027
Info : XDS110: connected to target via SWD
Info : XDS110: SWCLK set to 2500 kHz
Info : clock speed 8500 kHz
Info : SWD DPIDR 0x2ba01477
Info : [cc32xx.cpu] Cortex-M4 r0p1 processor detected
Info : [cc32xx.cpu] target has 6 breakpoints, 4 watchpoints
Info : starting gdb server for cc32xx.cpu on 3333
Info : Listening on port 3333 for gdb connections
    TargetName         Type       Endian TapName            State       
--  ------------------ ---------- ------ ------------------ ------------
 0* cc32xx.cpu         cortex_m   little cc32xx.cpu         running

Info : [cc32xx.cpu] requesting target halt and executing a soft reset
Error: Could not write to register 'msp'

Info : XDS110: disconnected
FATAL ERROR: command exited with status 1: /usr/local/bin/openocd -s /Users/eric/zephyrproject/zephyr/boards/ti/cc3220sf_launchxl/support -s /Users/eric/zephyr-sdk-0.16.5-1/sysroots/x86_64-pokysdk-linux/usr/share/openocd/scripts -f /Users/eric/zephyrproject/zephyr/boards/ti/cc3220sf_launchxl/support/openocd.cfg '-c init' '-c targets' -c 'reset init' -c 'flash write_image erase /Users/eric/zephyrproject/zephyr/build/zephyr/zephyr.hex' -c 'reset run' -c shutdown
```
